cfme/tests/services/test_provision_stack.py

```diff
@@ -6,7 +6,7 @@
 from cfme.services.catalogs.catalog import Catalog
 from cfme.services.catalogs.orchestration_template import OrchestrationTemplate
 from cfme.services.catalogs.service_catalogs import ServiceCatalogs
-from cfme.services.catalogs.myservice import MyService
+from cfme.services.myservice import MyService
```

Comment:
NOTE: I am not addressing test_provision_stack failures in this PR, see #3805 for changes to this test.